<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiZip</class>
 <widget class="QDialog" name="GuiZip">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Archivos Comprimidos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/img/iconos/portada.png</normaloff>:/img/iconos/portada.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>3.000000000000000</double>
  </property>
  <widget class="QPlainTextEdit" name="txtComentarios">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>28</y>
     <width>256</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>30</horstretch>
     <verstretch>30</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>382</width>
     <height>30</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>34</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Archivos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comentarios</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDescomprimirIndividual">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>409</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Descomprimir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnvisualizar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>409</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>visualizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDescomprimir">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>409</y>
     <width>96</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Descomprimir todo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAbrir">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>409</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>82</y>
     <width>581</width>
     <height>321</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QListWidget" name="jlArchivos">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
